integration-tests/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply from: "$rootProject.projectDir/base-application.gradle"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
We can probably use the `compileVersion` here? Not sure if we want to link it to the sdk, but I think that's ok?